### DISCUSSION TITLE
Relay mode/Mesh networking

### DISCUSSION THREAD

**Discussion by glory86** [svpcom/wfb-ng#227] [Ideas]
hello Vasily

1. In your opinion what is the most reliable way to implement relay networking in WFB to extend the range even further

2. or even better is it possible to implement mesh networking with dynamic node connect/disconnect and routing through the network 

the solution which i came up with for the first problem is to make one drone an air station with two running WFB instances 
one as a ground for the second drone 
and the other one as a drone for the real ground station

any advice, reference, guideline, etc would be very much appreciated

regards

**Comment by svpcom** [comment]
Hi,
This is very complex problem to make real WIFI mesh:
1. You need to coordinate medium access between adjacent nodes. I.e. manage frequencies and collision resolution (what WIFI access point does)
2. You need a truly distributed mesh routing like https://github.com/cjdelisle/cjdns
